pyracf/common/irrsmo00.py

```diff
@@ -59,52 +59,34 @@ def call_racf(
         running_userid = b""
         if run_as_userid:
             running_userid = run_as_userid.encode("cp1047")
-        response = self.__call_irrsmo00_wrapper(
-            request_xml,
-            len(request_xml),
-            self.__response_buffer_size,
-            irrsmo00_options,
-            running_userid,
-            len(running_userid),
+        result = call_irrsmo00(
+            request_xml=request_xml,
+            request_xml_length=len(request_xml),
+            result_buffer_size=self.__result_buffer_size,
+            irrsmo00_options=irrsmo00_options,
+            running_userid=running_userid,
+            running_userid_length=len(running_userid),
         )
         # Preserve raw binary respone just in case we need to create a dump.
```

Comment:
Typo: `respone` should be `response`

Reply:
This typo has been fixed. Actually, I have made the naming convention consistent by referring to this as the "raw result XML" instead of "raw binary response" for better clarity and consistency. I must have missed this one because of the typo. This should be good now.